modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -75,21 +75,21 @@ If issues occur after adding the certificates, verify the specified values at th
 
 .Verifying imported certificates at the {prod-short} instance level
 
-* In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/understanding/olm-what-operators-are.html[Operator] deployment, the namespace where the `CheCluster` is located contains labeled ConfigMaps with the correct content:
+* In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/understanding/olm-what-operators-are.html[Operator] deployment, the namespace where the `CheCluster` is located contains the labeled `ConfigMap` objects with the correct content:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get cm --selector=app.kubernetes.io/component=ca-bundle,app.kubernetes.io/part-of=che.eclipse.org -n {prod-namespace}
 ----
 +
-Check the content of ConfigMap by entering:
+Check the content of the `ConfigMap` object by entering:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get cm __<name>__ -n {prod-namespace} -o yaml
 ----
 
-* {prod-short} Pod Volumes list contains a volume that uses `ca-certs-merged` ConfigMap as data-source.
+* {prod-short} Pod Volumes list contains a volume that uses the `ca-certs-merged` `ConfigMap` object as the `data-source`.
```

Comment:
Why `data-source`?

Reply:
It's either `data-source` (if `data-source` appears on a yaml line) or data source.
@tolusha, which one is it?

Reply:
it should be: data source

Reply:
Thank you so much, Anatolii. Fixing.